### PR TITLE
Added log out authored by Ayman

### PR DESCRIPTION
**Pull request by Madonahs**
Log Out Added by Amy Khan.

### CODE REVIEW

File: app/src/main/java/com/madonasyombua/growwithgoogleteamproject/MainActivity.java [hunk 1]
```diff
@@ -184,18 +188,47 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
...
+                        //log out the user and launch LoginActivity
+                        Toast.makeText(MainActivity.this, "Inside Yes Dialog", Toast.LENGTH_LONG).show();
+                        MainActivity.this.finish();
+                        Intent intent = new Intent(MainActivity.this, LoginActivity.class);
+                        intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+                        intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK);
+                        intent.addFlags(Intent.FLAG_ACTIVITY_NO_ANIMATION);
+                        startActivity(intent);
+                        finish();
+                        System.exit(0);
```
Comment (CleverChuk) [inline app/src/main/java/com/madonasyombua/growwithgoogleteamproject/MainActivity.java]: When a user logs out, it is not reasonable to kill the app rather log them out and send them to sign in fragment.

Comment (Madonahs) [reply app/src/main/java/com/madonasyombua/growwithgoogleteamproject/MainActivity.java]: Noted will notify @aymkhan  and can also help her 👍

Comment (Madonahs) [reply app/src/main/java/com/madonasyombua/growwithgoogleteamproject/MainActivity.java]: I see.

Comment (MaherSoua) [reply app/src/main/java/com/madonasyombua/growwithgoogleteamproject/MainActivity.java]: also finish() is enough to quit app, but as @CleverChuk we have just to redirect them to login activity, also we have to update login status in SharedPreference like we do for login.

File: app/src/main/java/com/madonasyombua/growwithgoogleteamproject/MainActivity.java [hunk 2]
```diff
@@ -172,9 +176,9 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
                 // Take user to edit profile screen
                 return false;
 
-            case R.id.notifications:
-                // Take user to notification screen
-                return false;
+            //case R.id.notifications:
```
Comment (MaherSoua) [inline app/src/main/java/com/madonasyombua/growwithgoogleteamproject/MainActivity.java]: keep the case without comment

File: app/src/main/java/com/madonasyombua/growwithgoogleteamproject/MainActivity.java [hunk 3]
```diff
@@ -184,18 +188,47 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
...
             case R.id.about:
                 // Take user to about screen
+
                 return false;
             case R.id.help:
                 // Take user to help screen
                 return false;
             case R.id.logout:
                 // logout
+                final AlertDialog.Builder builder = new AlertDialog.Builder(this);
```
Comment (MaherSoua) [inline app/src/main/java/com/madonasyombua/growwithgoogleteamproject/MainActivity.java]: put all this staff in a method, may be we will call it in another place

**Comment by Madonahs** [comment]
@MaherSoua Yeah, lol thought it was set to default for now. 👍  nice.

**Comment by MaherSoua** [comment]
Please don't close this PR until take in consideration comments, we have to get clean code and nice behavior in the app, until now the logout kill app, it shouldn't

**Comment by Madonahs** [comment]
But she's still learning how to do it lol. So no need to open it. So i will tell her of the changes needed.

**Comment by MaherSoua** [comment]
The idea is not only learning how to code, but also learning how we use tools, in our case as we need some modification in this code, she will try to make changes, after that we can merge, and like that she can learn many things in one Pull Request :)

**Comment by Madonahs** [comment]
So i suggest her doing her changes and creating her own PR. So that will be fair. 😊

**Comment by MaherSoua** [comment]
@Madonahs why :-) as we have already a Pull Request? we can have a very nice history on our work, I assigned this PR to her.

**Comment by aymkhan** [comment]
Woops. Thanks for reassigning the task @MaherSoua . I'll go ahead and create a logout() method in LoginFragment and try calling the remove method on the username/password strings from SharedPreferences

**Comment by MaherSoua** [comment]
@aymkhan Thank you very much to help us build this project and yes feel free to add modification in the application, in all cases we all learn from each other and do review for each other :)

**Pull request closed without merging**